model/proto/api_v2/query.proto

```diff
@@ -109,10 +109,17 @@ message GetServicesResponse {
 
 message GetOperationsRequest {
   string service = 1;
+  string span_kind = 2;
+}
+
+message Operation {
+    string name = 1;
+    string span_kind = 2;
 }
 
 message GetOperationsResponse {
-  repeated string operations = 1;
+  repeated string operationNames = 1; //deprecated
```

Comment:
Please include an update to CHANGELOG, as this is a breaking change. I suggest also mentioning grpc plugin changes and the db schema changes / upgrade instructions.

If we are not populating operationNames field, we can rename it to `_`. But it might be better to populate both collections and have a config options to disable populating the names for installations that don't have legacy users and want a bit more efficiency. This way installation can be upgraded to a new version and clients can switch to the new API not in lock-step.

Reply:
I would prefer to rename it to _ if grpc server is not widely used, otherwise we will create tech debt for future cleanup (to remove the flag and code that generate operation names from the structs) and this usually ends up stay in code for months to years.

Reply:
Per offline discussion, we will set both operationNames and operations in the grpc response, and remove operationNames after 2+ releases